policy/modules/services/ssh.if

```diff
@@ -408,6 +408,10 @@ template(`ssh_role_template',`
 	allow $3 $1_ssh_agent_t:fifo_file rw_inherited_fifo_file_perms;
 	allow $3 $1_ssh_agent_t:process sigchld;
 
+	ifdef(`distro_debian', `
+		userdom_user_runtime_filetrans($1_ssh_agent_t, ssh_agent_tmp_t, sock_file, "openssh_agent")
```

Comment:
I missed this the first time, but the name (`"openssh_agent"`) doesn't seem necessary here.